<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1217</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project Xiso</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Qylon/Resources/Resources.qrc">
    <normaloff>:/Qylon/Resources/Icon.png</normaloff>:/Qylon/Resources/Icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1217</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionPanel"/>
    <addaction name="actionDebug_Console"/>
   </widget>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget_panel">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Settings</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="6">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Calibration</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="1" column="1">
            <widget class="QPushButton" name="pushButton_bright">
             <property name="text">
              <string>Bright</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButton_dark">
             <property name="text">
              <string>Dark</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QSpinBox" name="spinBox_calibCount">
             <property name="maximum">
              <number>100000</number>
             </property>
             <property name="value">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QPushButton" name="pushButton_out">
             <property name="text">
              <string>Out</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_Acquisition">
          <property name="title">
           <string>Acquisition</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QPushButton" name="pushButton_Sequential">
                 <property name="text">
                  <string>Sequential</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox_Sequential">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>100000</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_Continuous">
               <property name="text">
                <string>Continuous</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QGroupBox" name="groupBox_ROI">
          <property name="title">
           <string>ROI</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Width</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox_width">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>99999</number>
                 </property>
                 <property name="value">
                  <number>2400</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Height</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox_height">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>99999</number>
                 </property>
                 <property name="value">
                  <number>600</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="0" column="1">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_roiX">
                 <property name="text">
                  <string>X</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox_roiX">
                 <property name="maximum">
                  <number>99999</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_roiY">
                 <property name="text">
                  <string>Y</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBox_roiY">
                 <property name="maximum">
                  <number>99999</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QPushButton" name="pushButton_roiApply">
             <property name="text">
              <string>Apply</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Grabber Settings</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButton_GrabberInitialize">
             <property name="text">
              <string>Initialize</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButton_ResetGrabber">
             <property name="text">
              <string>Reset Grabber</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QGroupBox" name="groupBox_camera">
          <property name="title">
           <string>Detector Settings</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="3" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Binning Mode</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_BinningMode">
               <property name="currentText">
                <string>1X1</string>
               </property>
               <item>
                <property name="text">
                 <string>Unknown</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1X1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2X2</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>4X4</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_exposure">
               <property name="text">
                <string>Exposure Time</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_Exposure">
               <property name="maximum">
                <number>99999999</number>
               </property>
               <property name="value">
                <number>50</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButton_DetectorInitialize">
             <property name="text">
              <string>Initialize</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Exposure Mode</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="comboBox_exposureMode">
               <item>
                <property name="text">
                 <string>FPS25</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>FPS30</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>TRIG</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>SEQ</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>XFPS</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item row="4" column="0">
            <widget class="QPushButton" name="pushButton_detectorApply">
             <property name="text">
              <string>Apply</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_debug">
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Debug Console</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QGridLayout" name="gridLayout_9">
     <item row="0" column="0">
      <layout class="QFormLayout" name="formLayout"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionGrabber">
   <property name="text">
    <string>Grabber</string>
   </property>
  </action>
  <action name="actionDetector">
   <property name="text">
    <string>Detector</string>
   </property>
  </action>
  <action name="actionSequential_Grabbing">
   <property name="text">
    <string>Sequential Grabbing</string>
   </property>
  </action>
  <action name="actionContinuous_Grabbing">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Continuous Grabbing</string>
   </property>
  </action>
  <action name="actionPanel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Settings Panel</string>
   </property>
  </action>
  <action name="actionDebug_Console">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Debug Console</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Qylon/Resources/Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_Continuous</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButton_Sequential</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Continuous</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_Sequential</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Continuous</sender>
   <signal>clicked(bool)</signal>
   <receiver>pushButton_ResetGrabber</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
